web/templates/compose_control_buttons.hbs

```diff
@@ -1,36 +1,36 @@
 <div class="compose-control-buttons-container order-1">
     <input type="file" class="file_input notvisible" multiple />
     <div class="compose_control_button_container" data-tippy-content="{{t 'Preview' }}">
-        <a role="button" class="markdown_preview compose_control_button zulip-icon zulip-icon-preview" aria-label="{{t 'Preview' }}" tabindex=0></a>
+        <a role="button" id="compose_control_button" class="markdown_preview compose_control_button zulip-icon zulip-icon-preview" aria-label="{{t 'Preview' }}" tabindex=0></a>
```

Comment:
`id` values have to be unique; you can't reuse them like this.

Is there a reason you can't use the existing classes on the buttons to make this work? If it's a specificity issue, that probably means there needs to be some more work done in the CSS to untangle inheritance issues.

Reply:
Oh No, there wasn't any reason for that i'll make the necessary changes now

Reply:
I've further optimized the PR by leveraging existing CSS classes and IDs for button styles. I remove the `:focus ` pseudo class without replacing it with `:focus-within` but that's okay because the setting outline: revert; on compose interactive buttons using the existing classes covers for this.